ntoskrnl/inbv/inbv.c

```diff
@@ -1284,7 +1284,7 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
             /* Save previous screen pixels to buffer */
             InbvScreenToBufferBlt(Buffer, 0, 0, 22, 9, 24);
             /* Draw the progress bar bit */
-            InbvBitBlt(Bar, 0, 0);
+            BitBltPalette(Bar, TRUE, 0, 0);
```

Comment:
When we don't use the rotbar (as it should be by default), do we still get the blink?

Reply:
If you are talking about using with this condition
```
if (ROT_BAR_DEFAULT_MODE == RB_SQUARE_CELLS && Bar)
```
then we don't have any blink.

Blink happens only with InbvBitBlt, because it draws a palette of Bar image, then we doing fade animation.

Reply:
I also tested RB_SQUARE_CELLS, everything seems to be fine

Reply:
Oh yes indeed; and actually the whole mechanism was in place already ^^